mne/utils/docs.py

```diff
@@ -3674,6 +3674,14 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     If ``reject`` is ``None`` (default), no rejection is performed.
 """
 
+docdict[
+    "remove_dc"
+] = """
+remove_dc : bool
+    If ``True``, the mean is subtracted from each segment. If ``False``,
+    the data is left as is.
```

Comment:
```suggestion
    If ``True``, the mean is subtracted from each segment before
    computing its spectrum.
```

or something like that, feel free to tweak to clarify.

Reply:
ok will work on this

Reply:
I just put what you suggested